pkgs/by-name/so/soundalike/package.nix

```
@@ -0,0 +1,61 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitea,
+  chromaprint,
+  makeWrapper,
+}:
+
+buildGoModule rec {
+  pname = "soundalike";
+  version = "0.1.2";
+
+  src = fetchFromGitea {
+    domain = "codeberg.org";
+    owner = "derat";
+    repo = "soundalike";
+    tag = "v${version}";
+    hash = "sha256-mpYUVTj3Zll6kNuK5Mdzv1R7k5FZy6XFghhzmAPPVM8=";
+  };
+
+  vendorHash = "sha256-7hRezOBcjB2wsx/SwV519wg3Azh+0kHMcAoc9aYPM3A=";
+
```

```suggestion

  ldflags = [
    "-s"
    "-w"
    "-X"
    "main.buildVersion=${version}"
  ];
```
To mirror the upstream release script. Without the last part it prints this for `-version`:

```
soundalike version [non-release] compiled with go1.24.1 for linux/amd64
fpcalc version 1.5.1 (FFmpeg Lavc61.19.100 Lavf61.7.100 SwR5.3.100)
```

---

Updated in latest commit: https://github.com/NixOS/nixpkgs/pull/388332/commits/bd9ebda4bf20d7b53b76e1509aba7ceacfd9988e